src/backend/vulkan/src/window.rs

```diff
@@ -249,11 +249,16 @@ impl w::Surface<Backend> for Surface {
     fn capabilities(&self, physical_device: &PhysicalDevice) -> w::SurfaceCapabilities {
         // Capabilities
         let caps = unsafe {
-            self.raw
+            match self.raw
                 .functor
                 .get_physical_device_surface_capabilities(physical_device.handle, self.raw.handle)
-        }
-        .expect("Unable to query surface capabilities");
+                {
+                    Ok(caps) => caps,
+                    #[cfg(target_os = "android")]
+                    Err(vk::Result::ERROR_SURFACE_LOST_KHR) => vk::SurfaceCapabilitiesKHR::default(),
```

Comment:
It would be great to log some warning here. And in other places with same logic.